crates/forge-runner/src/running.rs

```diff
@@ -183,39 +202,48 @@ fn build_syscall_handler<'a>(
     )
 }
 
+#[derive(Debug)]
 pub struct RunResultWithInfo {
     pub(crate) run_result: Result<RunResult, RunnerError>,
     pub(crate) gas_used: u128,
 }
 
+// TODO merge this into test-collector's `TestCase`
```

Comment:
Please create an issue, this one is important.

Reply:
#1433